scripts/integration-test.sh

```diff
@@ -131,7 +131,7 @@ function cleanup() {
   echo "INFO: Terminating background jobs"
   echo "  any kill usage errors below may be ignored"
 
-  for p in collector processor middleware
+  for p in collector processor
```

Comment:
Mmm, I see. Damn, that happened to me before and it's a pain. I guess the easiest thing to do is to just remove it indeed. But then we should hurry and remove all of the middleware asap.